<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="select">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select a Query Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>0</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start!!!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load database</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Query Image:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_5">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_6">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_7">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>120</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_8">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>120</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_9">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>120</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_10">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>120</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_11">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>120</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_12">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>120</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_13">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_14">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_15">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_16">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_18">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>220</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_19">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>220</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_20">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>220</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_21">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>220</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_22">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>220</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_23">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>220</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_24">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>220</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_25">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_26">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>310</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_27">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_28">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_29">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>310</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_30">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>310</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_31">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>310</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_32">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>310</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_33">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>310</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_34">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>310</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_35">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>310</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_36">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>310</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_37">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>410</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_38">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>410</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_39">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>410</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_40">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>410</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_17">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>220</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_0">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>It might take 5 minutes to load database, be patient...</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
